src/admin/admin.controller.ts

```diff
@@ -16,48 +15,42 @@ export class AdminController {
     @Roles('Admin', 'school')
     @UseGuards(JwtAuthGuard)
     async fetchUsers(@Request() req, @Body() data: any): Promise<UsersResponse> {
-        const users: UsersResponse = await this.adminService.fetchUsers(data);
-        return users;
+        return await this.adminService.fetchUsers(data);
     }
 
     @Post('/changePassword')
     @Roles('Admin', 'school')
     @UseGuards(JwtAuthGuard)
     async updatePassword(@Body() data: {loginId: string, password: string}): Promise<SignupResponse> {
-        const status: SignupResponse = await this.adminService.updatePassword(data);
-        return status;
+        return await this.adminService.updatePassword(data);
```

Comment:
You can remove the await as well to make this cleaner. Do this for others as well.

Reply:
Fixed